evap/student/templates/student_vote.html

```diff
@@ -312,6 +315,117 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
             function textResultsPublishConfirmationBottomChanged() {
                 $("#text_results_publish_confirmation_top").prop('checked', $("#text_results_publish_confirmation_bottom").prop('checked'));
             }
+
+            function selectByNumberKey(row, num) {
+                if (num === 0) {
+                    // Select "No answer"
+                    num = row.children.length;
+                }
+                const index = num - 1;
+
+                if (!(0 <= index && index < row.children.length)) return;
+
+                const maybeNextElement = row
+                    .children[index]
+                    .getElementsByClassName("num-selectable");
+                if (maybeNextElement.length === 1) {
+                    maybeNextElement[0].click();
+                }
```

Comment:
This class is another safety guard to not click on the wrong things (although this method should only be called with safe options). I don't think the querySelector is worth it, because it is slower (only slightly) and with the current code, there is another safety that there really only is one (which should always be the case in the calling context). What do you think?

Reply:
@pixunil would you be fine with keeping the length check here?